tokio-console/src/view/mod.rs

```diff
@@ -207,6 +209,13 @@ impl View {
     }
 }
 
+pub(crate) fn dur(styles: &Styles, dur: std::time::Duration) -> Span<'static> {
```

Comment:
i don't love making this internal helper public when it's just a small wrapper around `styles.time_units()` and there's no doc comment explaining what this does and how it's distinct from just calling `styles.time_units()`. i think this was less of a problem when it was an internal helper in one module where it was only used in that module.

could we add a doc comment or a clearer name to this so that its use is more obvious?

Reply:
Since this is only used in 3 places, I inlined the call to `time_units` and removed this function completely.